### ISSUE TITLE
Bug: the ansible-output from one task shows in the another task output (different users)

### ISSUE THREAD

**Issue by metacoma** [ansible-semaphore/semaphore#322]
Steps to reproduce:
1. Create user **A**, create user **B**
2. Login in semaphore under user **A**, create project, inventory, environment, playbook repo, task template
3. Login in semaphore under user **B**, create project, inventory, environment, playbook repo, task template
4. Run task under user A (term **taskA**)
5. Run task under user B (term **taskB**)

ansible-output from **taskA** shows in output in **taskB** 

Version: tag v.2.2.0

**Comment by matejkramny** [comment]
Thanks, will investigate

**Comment by Natanande** [comment]
This is even more prevalent when running tasks concurrently after #366
